spec/Classes.tex

```diff
@@ -309,11 +309,14 @@ \subsection{Overriding Base Class Methods}
 dispatch in the event that a variable that has the base class type
 references an object that has the derived class type.
 
-The identical signature requires that the names, types, and order of
+The identical signature requires that the names, intents, types, and order of
 the formal arguments be identical. The return type of the overriding
 method must be the same as the return type of the base class's method,
 or must be a subclass of the base class method's return type.
```

Comment:
* drop comma before "or" or add "it" after the "or" (I think I prefer the latter)
* this might also parse more cleanly if "either" was inserted after the first "must" (in which case I'd drop the comma and the second "must"